template/web/src/components/SmartWallets/PaymasterClient.ts

```diff
@@ -1,7 +1,16 @@
-import { createPublicClient, http } from 'viem';
+import { ENTRYPOINT_ADDRESS_V06 } from 'permissionless';
+import { paymasterActionsEip7677 } from 'permissionless/experimental';
+import { createClient, createPublicClient, http } from 'viem';
 import { baseSepolia } from 'viem/chains';
 
+const paymasterService = process.env.NEXT_PUBLIC_PAYMASTER_URL ?? '';
+
 export const client = createPublicClient({
   chain: baseSepolia,
   transport: http(),
 });
+
```

Comment:
Why do we have two clients?

Maybe is fair to explain that.

Reply:
Also should this client shared across all experiences in BOAT?

Reply:
Ah thanks that is an unused client - removed

Reply:
This client is used in `willSponsor.test.ts`. I tried to define it directly in the test file but it doesn't work...getting type errors.

Reply:
> Also should this client shared across all experiences in BOAT?

I'm not sure. Can refactor if we see that this client is needed elsewhere?

Reply:
let's keep it for future refactor, either way seems something that can be reused.